src/js/crypto.js

```diff
@@ -104,58 +139,61 @@ window.App.Bitcoin = {
         const storageSetting =
             App.ENV.platform === 'EXTENSION' ? 'BROWSER_STORAGE' : 'LOCAL_STORAGE';
 
-        Object.keys(this.PERIODS).forEach((period) => {
-            this.repositories[period] = new SuperRepo({
+        ['bitcoin', 'ethereum'].forEach((cryptoType) => {
```

Comment:
That's kind of a two-sided performance problem. First, SuperRepo instances are created for both tokens (although only one could be active at a time), and second (more importantly) - the inactive one will still tick-tack and fire API calls for both - the currently active and the inactive tokens.